kmultiversx/src/kmultiversx/utils.py

```diff
@@ -66,6 +69,13 @@ def krun_config(krun: KRun, conf: KInner, pipe_stderr: bool = False) -> KInner:
     return krun.kore_to_kast(res_conf_kore)
 
 
+def llvm_interpret_raw(definition_dir: Path, kore_input: str, pipe_stderr: bool = False) -> CompletedProcess:
```

Comment:
IIUC, this is necessary so that the output is not parsed on status code 0. Is my understanding correct?

Reply:
Yes, `llvm_interpret` is using the kore parser on the output regardless of the exit code, which we don't need.